spec/src/modules/tracker.js

```diff
@@ -3938,6 +4219,7 @@ describe('ConstructorIO - Tracker', () => {
       tracker.on('success', (responseParams) => {
         const requestParams = helpers.extractBodyParamsFromFetch(fetchSpy);
 
+        try {
```

Comment:
Is this code in this try block missing indentations?

Also, what prompted adding the try/catch blocks? 🤔 Do we need them?

Reply:
I was debugging tests, for some reason my debugger wasn't working. 

So, I added try/catch so that the tests don't hang and also print the error. The way our tests are setup right now if assertions fail then the test just times out since done() never gets called.